cmd/keymasterd/certgen.go

```diff
@@ -21,6 +21,17 @@ import (
 
 const certgenPath = "/certgen/"
 
+func prependGroups(groups []string, prefix string) []string {
```

Comment:
what is this prependGroup For? gitDB issues?

Reply:
This is the first part of supporting separate "namespaces" for different data sources. For example, I'll be adding support for Okta groups, but they can be prefixed.